graph/src/data/schema.rs

```diff
@@ -862,6 +862,7 @@ impl Schema {
                     .find(|field| {
                         name == &field.name.as_str().to_camel_case()
                             || name == &field.name.to_plural().to_camel_case()
+                            || field.name.eq(name)
```

Comment:
Ok, let's go with this for now, instead of a `_fulltext` prefix in the column names in Postgres. We can still lift this "limitation" of not being allowed to name fulltext resolvers the same as entity field names later, if we want.